Dockerfile

```diff
@@ -35,4 +35,13 @@ ENV ELASTIC_APM_USE_STRUCTLOG=True
 
 EXPOSE 3000
 
+LABEL \
+    org.label-schema.schema-version="1.0" \
+    org.label-schema.vendor="Elastic" \
+    org.label-schema.name="opbeans-python" \
+    org.label-schema.version="5.7.0" \
```

Comment:
We just released 5.8.0 yesterday, so would probably make sense to reflect that here

Reply:
It should be bumped automatically, I'll review the process and bump it in another PR with the agent dependency, does it sound good?